The type 'IsExternalInit' exists in both AntDesign and System.Runtime

### Describe the bug
Cannot build project after upgrading nuget packages (last .asp NET core libs & ant blazor)

This is error what I got:
`error CS0433: The type 'IsExternalInit' exists in both 'AntDesign, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null' and 'System.Runtime, Version=8.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'`

### Steps to reproduce (please include code)
Last stable version of dotnet & ant blazor.

We will close this issue if:
Successfully build project

### Exceptions (if any)

`The type 'IsExternalInit' exists in both 'AntDesign, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null' and 'System.Runtime, Version=8.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'`


May be related (we use http polly). And seems like they use PolySharp which may face same error.

https://github.com/Sergio0694/PolySharp/issues/18

--- a/components/IsExternalInit.cs
+++ b/components/IsExternalInit.cs
@@ -1,6 +1,6 @@
 ﻿namespace System.Runtime.CompilerServices
 {
-    public sealed class IsExternalInit
+    internal sealed class IsExternalInit
     {
     }
 }

--- a/site/AntDesign.Docs/IsExternalInit.cs
+++ b/site/AntDesign.Docs/IsExternalInit.cs
@@ -4,7 +4,7 @@
     /// 解决使用record类型时发生的下面错误
     /// 错误 CS0518  预定义类型“System.Runtime.CompilerServices.IsExternalInit”未定义或导入 AntDesign.Docs(netcoreapp3.1)
     /// </summary>
-    public sealed class IsExternalInit
+    internal sealed class IsExternalInit
     {
     }
 }

